src/ProxyManager/GeneratorStrategy/EvaluatingGeneratorStrategy.php

```diff
@@ -43,11 +40,9 @@ public function generate(ClassGenerator $classGenerator) : string
 
         // @codeCoverageIgnoreStart
         if (! $this->canEval) {
-            $fileName = tempnam(sys_get_temp_dir(), 'EvaluatingGeneratorStrategy.php.tmp.');
+            $fileName = __DIR__ . DIRECTORY_SEPARATOR . 'EvaluatingGeneratorStrategy.php.tmp';
```

Comment:
Let's avoid relying on `DIRECTORY_SEPARATOR`: a `/` is more consistent and readable too

Reply:
Done now - after your approval